web/src/components/Layout/CollectionList.tsx

```diff
@@ -80,8 +80,9 @@ export default function CollectionList({
                 variants={transitionVariants}
                 initial={"fadeOut"}
                 animate={"fadeIn"}
+                className={"whitespace-nowrap"}
               >
-                Nothing to display &#128542;
+                {"No collections to show 😔"}
```

Comment:
I'm using a string here because of the emoji. I could use the Unicode before but it is a pain having to look those up every time and then we can't see which emoji we are using in the source code